components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -697,7 +728,7 @@ public User updateUser(User user, Map<String, Boolean> requiredAttributes) throw
      * @throws CharonException Error while filtering
      */
     private List<Object> filterUsersBySingleAttribute(ExpressionNode node, Map<String, Boolean> requiredAttributes,
-            int offset, int limit, String sortBy, String sortOrder, String domainName) throws CharonException {
+            int offset, Integer limit, String sortBy, String sortOrder, String domainName) throws CharonException {
```

Comment:
Is it necessary to change the signature here

Reply:
fixed: 8f8c20292c4fac76778a8aa95490675023d7d063